lib/mix_test_watch/runner.ex

```diff
@@ -2,6 +2,8 @@ defmodule MixTestWatch.Runner do
   @moduledoc false
 
   alias MixTestWatch.Config
+  
+  @clear_ansi_escape_sequence "\x1bc"
```

Comment:
This didn't work for me on macOS in iTerm2 until I changed it to `\e[2J\e[3J\e[H`.

Reply:
Strange 🤔 maybe this is one of those things that is terminal emulator dependent?